Add model & permission changes for the LoS extension

jsangmeister opened the pull request:
Reordered some fields in `organization` to have settings always above relations

r-peschke commented:
Why did you reordered the fields? This will complicate the transformation to the relational fields version of the models.yml

r-peschke reviewed (changes requested):
Please revert the reordering of fields

jsangmeister commented:
I don't think we should keep the wrong order or that it hinders the relational database migration, but okay

jsangmeister commented:
I also included your comments via email @r-peschke

jsangmeister commented:
@r-peschke @rrenkert I merged `meeting/list_of_speakers_enable_structure_level_countdowns` into `meeting/list_of_speakers_default_structure_level_time` so that a time of 0 disables countdowns. This is possible again since there is no orga-wide default any more and it is consistent with the intervention feature. Please check.

I also added a check that the group permissions are the same as the ones defined in the `permissions.yml` as it was possible until now that the two diverge without anyone noticing. I don't think it is even a good idea to keep this information redundant and that we should check the valid permissions not via the model enum, but I refrained from changing this for now to reduce the implementation work and focus on the new features.

r-peschke reviewed (commented):
What's about the relation personal_note <-> meetig_user?

jsangmeister commented:
I did not change anything about the relation. I just added the `equal_fields` (which should have been there from the start and were probably overlooked - all relations in `meeting_user` should only point to models of the same meeting, obviously). This lead to a few minor errors in the tests, which I fixed. What should be removed there?

jsangmeister commented on openslides_backend/action/actions/personal_note/create.py:
```diff
@@ -64,7 +51,6 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
                         "user_id": self.user_id,
                         "meeting_id": instance["meeting_id"],
-                        "personal_note_ids": [instance["id"]],
                     }
                 ],
```
I just cleaned up this code part. Previously, the relation was set via the `meeting_user` actions, which was error-prone and redundant. Now, the relation is only set by setting the `meeting_user_id` in the personal note.

r-peschke replied:
Sorry, I overlooked that the personal_note will be set in meeting_user by the relation handling and is not needed to set it explicitly.

jsangmeister replied:
okay, no problem

r-peschke commented:
> I did not change anything about the relation. I just added the `equal_fields` (which should have been there from the start and were probably overlooked - all relations in `meeting_user` should only point to models of the same meeting, obviously). This lead to a few minor errors in the tests, which I fixed. What should be removed there?

Nothing else. It was just about the personal_note, see above.
